### PR TITLE
Lesson_9_Homework

### PR DESCRIPTION
**Pull request by VictorPop84**
Again, sorry for the late ping.

### CODE REVIEW

File: lesson_9/pop_victor/tema9-OOP1-VictorPOP.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+class User(object):
+
+    def __init__(self, nickname, real_name, age, country, join_year):
+        self.nickname = nickname
+        self.real_name = real_name
+        self.age = age
+        self.country = country
+        self.join_year = join_year
+
+    def __repr__(self):
```
Comment (Liitle) [inline lesson_9/pop_victor/tema9-OOP1-VictorPOP.py:23]: Nice, I like it !

File: lesson_9/pop_victor/tema9-OOP1-VictorPOP.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    def account_restriction(self):
+        company_country = ['Romania', 'Luxembourg', 'Austria', 'Germany']
+        if self.country in company_country:
+            return f'{self.nickname} has access to {self.country} Server'
+        return f'no server in {self.country} at the moment, sorry {self.nickname}'
+
+
+class Gamer(User):
+    def __init__(self, nickname, real_name, age, country, join_year, *args):
+        self.games = args
```
Comment (Liitle) [inline lesson_9/pop_victor/tema9-OOP1-VictorPOP.py:41]: Nice use of *ags. An alternative can be to ask for a list and set it as `[]` if the attribute is not specified:

```
def __init__(self, nickname, real_name, age, country, join_year, games=None)
    if games is None:
        self.games = []
    else:
         self.games = games
    super().__init__(nickname, real_name, age, country, join_year)
```

**Review by Liitle — APPROVED** [review]
Nice work, the code is clean and to the point. 

The changes that I mentioned about *args are optional. Let me know if you want to make the changes.

File: lesson_9/pop_victor/tema9-OOP1-VictorPOP.py (hunk 2 above)
Comment (VictorPop84) [reply lesson_9/pop_victor/tema9-OOP1-VictorPOP.py:41]: the use case was that you have a list of gamers.
thanks for the suggestion I will modify local but I would like to leave it as it is on git.

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**